readme.md

```diff
@@ -19,7 +19,7 @@ Create an app that demonstrates your subject. It should...
 
 ### Document It
 
```

Comment:
should we put the `Grouping` section higher up? that way, here, you can specify that they only need to submit one repo.